### DISCUSSION TITLE
DialogHost background fade out is to slow when using MaterialDesignEmbeddedDialogHost and TransitionAssist.DisableTransitions="True"

### DISCUSSION THREAD

**Discussion by poralcode** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2827] [Q&A] [answered]
I'd like to disable the transition of the `DialogHost` and at the same time, use the style `MaterialDesignEmbeddedDialogHost`.

```
<md:DialogHost
                Grid.Row="1"
                md:TransitionAssist.DisableTransitions="True"
                Identifier="ContainerDialogHost"
                SnackbarMessageQueue="{Binding ElementName=MySnackbar, Path=MessageQueue}"
                Style="{StaticResource MaterialDesignEmbeddedDialogHost}">

</md:DialogHost>
```

Together with `md:TransitionAssist.DisableTransitions="True"` and `MaterialDesignEmbeddedDialogHost` , opening the dialog is faster. However, upon closing it, I noticed that the `Background` is slowly fading out. Are there any other settings I am missing? There also should be no animation whatsoever upon closing it.

### Version

using version 4.5.0

**Comment by nicolaihenriksen** [comment, marked as answer]
This was also added as an issue #2826 which has been fixed.
